### PR TITLE
Feature/#486 merge integration tests

### LINKED ISSUE #486: merge integration test
Add the separate integration test and make them run against development.

### PR DESCRIPTION
**Pull request by hixi**
Reviewed by:
- [x] @das-g 

I wasn't able to use PhantomJS for testing this headless, these test need to be run manually for now!

Update: All integration tests passed on my machine.

Closes #486

### CODE REVIEW

File: docs/testing.md [hunk 1]
```diff
@@ -45,11 +45,39 @@ To run tests of all available test types, call the script without passing any op
...
+## Running selenium tests
+
+Prerequisite: a user login with username `admin` and password `admin`. To create this, run the command below:
+
+If haven't already, `source ./activate_local_development`.
+
+```bash
+docker-compose -f docker-compose-tests.yml run --rm frontend python3 web_frontend/manage.py createsuperuser
+```
+Enter `admin` as
```
Comment (das-g) [inline docs/testing.md]: The end of the sentence seems to be missing.

File: docs/testing.md [hunk 2]
```diff
@@ -45,11 +45,39 @@ To run tests of all available test types, call the script without passing any op
...
 ```shell
-docker-compose run webapp bash -c "DJANGO_SETTINGS_MODULE=config.settings.test coverage run --source='.' manage.py test;coverage html"
-``` 
+./runtests.py --driver Firefox
+```
+
+If you don't want to have Firefox running in front, under Linux there is a utility which helps in running
+this in a separate display: https://github.com/jordansissel/xdotool/blob/master/t/ephemeral-x.sh.
+
+This lets you execute any command in a separate display, and still getting the console output.
```
Comment (das-g) [inline docs/testing.md]: :+1:

File: local_docker_compose [hunk 3]
```diff
@@ -1,2 +0,0 @@
-#!/bin/bash
-docker-compose -f docker-compose.yml -f docker-compose-dev.yml "${@}"
```
Comment (das-g) [inline local_docker_compose]: :+1: for DRY w.r.t. ` activate_local_development`

File: docs/testing.md (hunk 1 above)
Comment (hixi) [inline docs/testing.md]: thanks, probably wanted to write something there, but what was it? ;-)

**Comment by hixi** [comment]
@das-g OK to be merged?

**Pull request merged**